nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -10,16 +10,43 @@ Apache Software License v2
 
 The following binary components are provided under the Apache Software License v2
 
-  (ASLv2) Groovy (org.codehaus.groovy:groovy:jar:2.4.5 - http://www.groovy-lang.org)
-  The following NOTICE information applies:
+  (ASLv2) Groovy (org.codehaus.groovy:groovy-all:pom:2.5.4 and its submodules included as transitive dependencies - http://www.groovy-lang.org)
```

Comment:
Some of the transitive dependencies appear to be missing.

Reply:
Please see [this comment](https://github.com/apache/nifi/pull/3547#discussion_r300889320)